include/rmm/mr/host/pinned_memory_resource.hpp

```diff
@@ -15,8 +15,8 @@
  */
 #pragma once
 
-#include "detail/error.hpp"
 #include "host_memory_resource.hpp"
+#include "rmm/detail/error.hpp"
```

Comment:
```suggestion
#include <rmm/mr/host/host_memory_resource.hpp>
#include <rmm/detail/error.hpp>
```

Reply:
Done, too. (And then clang-format swapped the order of these two).